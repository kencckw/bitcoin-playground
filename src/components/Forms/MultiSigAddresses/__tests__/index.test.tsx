import React from 'react';
import renderer from 'react-test-renderer';
import { MultiSigAddressesForm as Component } from '../';

it('renders correctly', () => {
  const tree = renderer.create(<Component />).toJSON();
  expect(tree).toMatchSnapshot();
});
