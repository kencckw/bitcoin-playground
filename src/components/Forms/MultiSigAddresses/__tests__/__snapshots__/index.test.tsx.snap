// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="ant-row ant-row-center ant-row-top"
  style={
    Object {
      "paddingTop": 12,
    }
  }
>
  <div
    className="ant-col ant-col-sm-24 ant-col-lg-16 ant-col-xl-12"
    style={
      Object {
        "width": "100%",
      }
    }
  >
    <div
      className="ant-card ant-card-bordered"
      style={
        Object {
          "minHeight": 650,
        }
      }
    >
      <div
        className="ant-card-body"
        style={Object {}}
      >
        <form
          className="ant-form ant-form-horizontal"
          onReset={[Function]}
          onSubmit={[Function]}
        >
          <div
            className="ant-page-header ant-page-header-ghost"
            style={
              Object {
                "paddingLeft": 0,
              }
            }
          >
            <div
              className="ant-page-header-heading"
            >
              <div
                className="ant-page-header-heading-left"
              >
                <span
                  className="ant-page-header-heading-title"
                  title="Multi-sig P2SH Bitcoin Address"
                >
                  Multi-sig P2SH Bitcoin Address
                </span>
              </div>
            </div>
          </div>
          <div
            className="ant-row ant-form-item"
            style={Object {}}
          >
            <div
              className="ant-col ant-form-item-label"
              style={Object {}}
            >
              <label
                className=""
                htmlFor="address"
                title="Generated address"
              >
                Generated address
              </label>
            </div>
            <div
              className="ant-col ant-form-item-control"
              style={Object {}}
            >
              <div
                className="ant-form-item-control-input"
              >
                <div
                  className="ant-form-item-control-input-content"
                >
                  <input
                    className="ant-input ant-input-disabled"
                    disabled={true}
                    id="address"
                    onBlur={[Function]}
                    onChange={[Function]}
                    onFocus={[Function]}
                    onKeyDown={[Function]}
                    type="text"
                    value=""
                  />
                </div>
              </div>
            </div>
          </div>
          <div
            className="ant-row ant-form-item"
            style={Object {}}
          >
            <div
              className="ant-col ant-form-item-label"
              style={Object {}}
            >
              <label
                className=""
                htmlFor="n"
                title="n"
              >
                n
              </label>
            </div>
            <div
              className="ant-col ant-form-item-control"
              style={Object {}}
            >
              <div
                className="ant-form-item-control-input"
              >
                <div
                  className="ant-form-item-control-input-content"
                >
                  <input
                    autoComplete="off"
                    className="ant-input"
                    id="n"
                    onBlur={[Function]}
                    onChange={[Function]}
                    onFocus={[Function]}
                    onKeyDown={[Function]}
                    type="text"
                    value=""
                  />
                </div>
              </div>
            </div>
          </div>
          <div
            className="ant-row ant-form-item"
            style={Object {}}
          >
            <div
              className="ant-col ant-form-item-label"
              style={Object {}}
            >
              <label
                className=""
                htmlFor="m"
                title="m"
              >
                m
              </label>
            </div>
            <div
              className="ant-col ant-form-item-control"
              style={Object {}}
            >
              <div
                className="ant-form-item-control-input"
              >
                <div
                  className="ant-form-item-control-input-content"
                >
                  <input
                    autoComplete="off"
                    className="ant-input"
                    id="m"
                    onBlur={[Function]}
                    onChange={[Function]}
                    onFocus={[Function]}
                    onKeyDown={[Function]}
                    type="text"
                    value=""
                  />
                </div>
              </div>
            </div>
          </div>
        </form>
      </div>
    </div>
  </div>
</div>
`;
